### ISSUE TITLE
Proxy option

### ISSUE THREAD

**Issue by fullzero87** [felipebergamin/api-telegram-bot#1]
In my country telegram website, including api, is blocked by government, so I have to use proxy. It would be nice to have such option only for bot without having to do it globally for request module.

**Comment by felipebergamin** [comment]
Hey, I haven't maintained this library for a few years, but now I'm picking it up again.

I've just updated the library, and it is now possible to provide a custom HTTP client for making requests to the Telegram API.
